src/Cli/problem.py

```diff
@@ -6,14 +6,18 @@
 import click
 import os
 from ..Scrapers.CFScraper import CFScraper
+from ..Scrapers.CCScraper import CCScraper
 from ..Problem.ProblemManager import ProblemManager
 
 @click.command()
 @click.argument('url', type=str)
 @click.argument('dest', type=str, default='.')
 def problem(dest, url):
     try:
-        scraper = CFScraper(url)
+        if url.find("codechef.com") != -1:
+            scraper = CCScraper(url)
+        elif url.find("codeforces.com") != -1:
+            scraper = CFScraper(url)
```

Comment:
Now, since we have more than 1 website support, we can create an exception whenever an unsupported website url given as input. UnsupportedPlatform seems a good option.